packages/react-core/src/demos/examples/HelperText/HelperTextStaticVariantStaticText.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Form, FormGroup, TextInput, HelperText, HelperTextItem } from '@patternfly/react-core';
+
+export const HelperTextStaticVariantStaticText: React.FunctionComponent = () => {
+  const [value, setValue] = React.useState('');
+
+  const handleInputChange = (inputValue: string) => {
+    setValue(inputValue);
+  };
+
+  return (
+    <Form>
+      <FormGroup label="Middle Name" fieldId="login-input-helper-text1">
+        <TextInput
+          type="text"
+          id="login-input-helper-text1"
+          name="login-input-helper-text1"
+          onChange={handleInputChange}
+          aria-describedby="helper-text1"
+          value={value}
+        />
+      </FormGroup>
+      <HelperText id="helper-text1">
```

Comment:
Can you move this up after the `<TextInput>` and wrap it in `<FormHelperText>` like in the [password strength helper text](https://github.com/patternfly/patternfly-react/blame/main/packages/react-core/src/demos/PasswordStrength.md#L153-L165)? That should result in less space between the text input and helper text.